app/controllers/comments_controller.rb

```diff
@@ -154,8 +162,7 @@ def thread
   end
 
   def thread_followers
-    return not_found unless @comment_thread.can_access?(current_user)
-    return not_found unless current_user.is_moderator || current_user.is_admin
+    return not_found unless @comment_thread.can_access?(current_user) && current_user.is_moderator
```

Comment:
Are there comments that a moderator cannot access? Is a private thread only accessible to the moderator that started it (not to other moderators)?

Otherwise the check for `can_access` seems redundant.

Reply:
A private thread should be available to all moderators (and admins), not just the moderator who created it.

Reply:
That does seem to be the intention elsewhere in the code, so I'm guessing that this should say "the current user can access OR the current user is a moderator" rather than "the current user can access AND the current user is a moderator". I'll go ahead and change it and we can double check that the behaviour matches once it's on the dev server.

Reply:
(Note that the removal of `is_admin` here is because all admins count as moderators, so `is_moderator` already includes all admins - admins are not being excluded.)

Reply:
I've had a closer look at this and I was mistaken about this being access to a comment. This is for access to a list of people who are following a comment thread. I'm assuming this should only ever be shown to moderators (including admins), so I'm just going to remove the check for whether the user can access, and simply restrict to only moderators.